esp/logging/logginglib/loggingagentbase.cpp

```diff
@@ -294,8 +294,8 @@ void CLogContentFilter::filterLogContent(IEspUpdateLogRequestWrap* req)
     StringBuffer updateLogRequestXML;
     toXML(updateLogRequestTree, updateLogRequestXML);
     ESPLOG(LogMax, "filtered content and option: <%s>", updateLogRequestXML.str());
-    req->clearOriginalContent();
-    req->setUpdateLogRequest(updateLogRequestXML.str());
+
+    return new CUpdateLogRequestWrap(req->getGUID(), req->getOption(), updateLogRequestXML.str());
```

Comment:
It doesn't appear that CLogThread::queueLog takes ownership of the new CUpdatelogRequestWrap when it calls filterLogContent. Is this a possible leak, or am I missing something?